carbonmark-api/helpers/formatUSDC.js

```diff
@@ -0,0 +1,6 @@
+import { utils } from "ethers";
+
+export const formatUSDC = (str) => {
+  const decimals = process.env.VERCEL_ENV === "development" ? 18 : 6;
```

Comment:
Why is it that we would want 18 decimals in development anyway?

Reply:
Najada distributed test "USDC" tokens on Mumbai from another project but they are 18 decimal :(

If we want to go all-in on testnet QA then we should create a 6 decimal one and use that instead. but its high effort low reward rn as testnet is always a pain (there are no carbon pools on testnet)